mlflow/utils/file_utils.py

```diff
@@ -762,14 +771,40 @@ def run_download(chunk: _Chunk):
             chunk = futures[future]
             try:
                 future.result()
-            except Exception:
+            except Exception as e:
+                _logger.warning(
```

Comment:
the warning looks like this:

<img width="806" alt="image" src="https://github.com/mlflow/mlflow/assets/17039389/3f7cea96-6743-4959-af74-2f8824f9998e">

this might annoy users

Reply:
If it does, we can use `debug`.

Reply:
I think debug might be better. It still gives us the ability to troubleshoot a failure, but users don't typically see transfer retries in Spark. Might as well keep it consistent, I feel.

Reply:
I don't feel this is a blocker, though.

Reply:
I'll file a followup PR to change the logging level. Merging this PR.